### PR TITLE
Create test user at start of selenium tests execution

### PR DESCRIPTION
**Pull request by dmytro-ndp**
### What does this PR do?
Implements creation of test user into `webdriver.sh` script of selenium tests.

### What issues does this PR fix or reference?
#7816

#### Docs PR
https://github.com/eclipse/che-docs/pull/328

### CODE REVIEW

**Comment by dmytro-ndp** [comment]
ci-test

**Comment by dmytro-ndp** [comment]
ci-test

**Review by tolusha — APPROVED** [review]

**Comment by dmytro-ndp** [comment]
Test report for Eclipse Che Multi User: https://ci.codenvycorp.com/view/qa-experimental/job/experimental-che-multiuser-integration-tests/15/#showFailuresLink

File: selenium/che-selenium-core/bin/webdriver.sh [hunk 1]
```diff
@@ -820,6 +821,65 @@ prepareToFirstRun() {
...
+    fi
+
+    # create test user by executing kcadm.sh commands inside keycloak docker container if there are no its credentials among environment variables
+    if [[ "${PRODUCT_HOST}" == "$(detectDockerInterfaceIp)" ]] || [[ "${CHE_SELENIUM_INFRASTRUCTURE}" == "openshift" ]]; then
+        local time=$(date +%s)
+        export CHE_TESTUSER_NAME=${CHE_TESTUSER_NAME:-user${time}}
+        export CHE_TESTUSER_EMAIL=${CHE_TESTUSER_EMAIL:-${CHE_TESTUSER_NAME}@1.com}
+        export CHE_TESTUSER_PASSWORD=${CHE_TESTUSER_PASSWORD:-${time}}
+
+        if [[ "${CHE_SELENIUM_INFRASTRUCTURE}" == "openshift" ]]; then
```
Comment (riuvshin) [inline selenium/che-selenium-core/bin/webdriver.sh]: `CHE_SELENIUM_INFRASTRUCTURE` was used only to use specific recipe for tests against che on openshift but now we don't need it anymore, but we need some flag which will allow us to do other some infra related things like specific test for che on OS. So can we rename `CHE_SELENIUM_INFRASTRUCTURE` => `CHE_INFRASTRUCTURE` and make sure that this property will allow us to do some infra specific actions.

**Review by riuvshin — APPROVED** [review]

**Pull request merged**